### PR TITLE
Support multiple sort options

### PR DESCRIPTION
**Pull request by mtlewandowsk89**
Before submitting this PR:

1. Make sure you are PR'ing from a fork, please do not push branches directly
   to this repository.
2. Ensure you've written sufficient tests for your work.
3. Double check that your changes will not break existing connectors (if
   applicable).
4. Double check that your changes do not break the `react-search-ui-views`
   storybook (if applicable).

These changes give the user the ability to sort by multiple different properties. The second and any additional properties will be used as tie breakers.

Allow search-ui to have multiple sort parameters.

https://github.com/elastic/search-ui/issues/552

### CODE REVIEW

**Comment by mtlewandowsk89** [comment]
Let me know if this circleci error is something on my end I need to fix or not. It said something about it being caused by the CI environment potentially.

File: examples/sandbox/src/App.js [hunk 1]
```diff
@@ -33,6 +33,28 @@ const SORT_OPTIONS = [
...
+    value: "states",
+    direction: "asc"
+  },
+  {
+    name: "State -> Title",
+    value: [
+      { "states" : "asc" },
+      { "title" : "asc" }
+    ],
+    direction: ""
```
Comment (ckeboss) [inline examples/sandbox/src/App.js:48]: This seems redundant if the direction is already passed in the `value`

File: packages/react-search-ui/src/containers/Sorting.js [hunk 2]
```diff
@@ -2,24 +2,45 @@ import PropTypes from "prop-types";
...
-  );
+function findSortOption(sortOptions, sortData) {
+  let value;
+  let direction;
+  if (Array.isArray(sortData)) {
+    value = sortData;
+
+    // We don't care about option.direction in this case since direction is part of the value array.
+    return sortOptions.find(
+      option => JSON.stringify(option.value) === JSON.stringify(value)
```
Comment (ckeboss) [inline packages/react-search-ui/src/containers/Sorting.js:17]: Can you explain the reasoning behind stringifying these objects and then comparing the strings?

**Review by ckeboss — COMMENTED** [review]
It also looks like the selected option is not selecting the correct value from the url state

File: packages/react-search-ui/src/containers/Sorting.js (hunk 2 above)
Comment (mtlewandowsk89) [reply packages/react-search-ui/src/containers/Sorting.js:17]: It didn't work without JSON.stringify. The find method was returning undefined until I added the JSON.stringify.

**Comment by mtlewandowsk89** [comment]
"It also looks like the selected option is not selecting the correct value from the url state". I will look into this. I didn't notice that at first.

**Comment by JasonStoltz** [comment]
Thank you both!

**Comment by martynball** [comment]
@ckeboss Awesome thanks!

**Pull request closed without merging**